apps/site/components/Containers/NavBar/index.module.css

```diff
@@ -28,12 +28,21 @@
     dark:border-neutral-900;
 }
 
+.sidebarItemToggler {
+  position: absolute;
```

Comment:
Hey @aduh95 we use Tailwind for the Styling of the website. Would you like me to change these lines? It is all documented on the CONTRIBUTING.md file with all instructions on how to do the styling <3 -- Feel free to also do it yourself if you wish.